lua/apg/modules/lag_detection.lua

```diff
@@ -32,8 +32,8 @@ local mod = "lag_detection"
 ]]----------------------
 
 local lagFix = {
-    cleanup_all = function( notify ) APG.cleanUp( "all", notify ) end,
-    cleanup_unfrozen = function( notify ) APG.cleanUp( "unfrozen", notify ) end,
+    cleanup_all = function( notify ) APG.cleanUp( "all" ) end,

```

Comment:
If we're not using `notify` in here anymore, we could remove the parameter for it too, yeah?

Reply:
Fixed